src/containers/Organization/Onboarding/Form.test.helper.ts

```
@@ -0,0 +1,87 @@
+    designation: 'finance',
+    phone: '09421050449',
+    email: 'finance@email.com',
+    billing_frequency: 'Annually',
+    name: 'finance poc firstName finance poc firstName',
+  },
+  registration_id: 1,
+  org_id: 2,
+  has_submitted: false,
+  billing_frequency: 'Annually',
+};
+export const setRegistrationData = (activeStep: Number) => {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)